src/game/aggregation.rs

```diff
@@ -0,0 +1,561 @@
+use std::{
+    collections::HashMap,
+    fmt::Display,
+    fs::{self, File},
+    io::Write,
+    path::Path,
+};
+
+use crate::{
+    card_to_string, flop_to_string,
+    utils::stats::{get_action_evs, get_action_frequencies, get_player_stats},
+    Action, ActionTree, PostFlopGame, TreeConfig,
+};
+
+// TODO do we ever realistically want the Skip option?
+// TODO also, for `Error`, we may actually want to error if the directory exists at all
+//      (as opposed to what's done now, which may not error until after some reports are already written)
+/// Describes possible behaviors for writing reports to files that already exist.
+#[derive(Clone, Copy)]
+pub enum ExistingReportBehavior {
```

Comment:
@bkushigian Thoughts on using this enum to allow uses to select the output behavior when a report directory already exists?

Reply:
Sounds good to me